### DISCUSSION TITLE
Welcome to bluecherry-apps Discussions!

### DISCUSSION THREAD

**Discussion by curtishall** [bluecherrydvr/bluecherry-apps#415] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 2206bank** [comment]
Hey There.....I am a little new to this ---- git hub, forums, etc, so please excuse signs of stupidity or dementia!?!!.....I am involved with image processing and have some command line experience.  We had a DVR based on BC built but am having some issues connecting other than IP cameras (SD Analog/USB primarily).   Is this a pipedream?  Anyone have success or failure accomplishing this?  Appreciate any feedback!

Len C

**Comment by haadiali72** [comment]
dpkg: error processing package bluecherry (--install):
 installed bluecherry package post-installation script subprocess returned error exit status 1
Processing triggers for libc-bin (2.31-0ubuntu9.9) ...
Processing triggers for systemd (245.4-4ubuntu3.22) ...
Processing triggers for rsyslog (8.2001.0-1ubuntu1.3) ...
Errors were encountered while processing:
 bluecherry

Any solution for this right after I configured the mysql server?
